### PR TITLE
indexer: Remove

### PR DESCRIPTION
**Pull request by mslipper**

### CODE REVIEW

File: .github/CODEOWNERS
Comment (sebastianst) [inline .github/CODEOWNERS:1]: @mslipper need to remove Josh, Github currently reports this file as errorneous

File: ops/scripts/update-op-geth.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 
 def main():
-	for project in ('.', 'indexer'):
+	for project in ('.',):
```
Comment (sebastianst) [inline ops/scripts/update-op-geth.py:13]:
```suggestion
	for project in ('.'):
```
or just remove the loop completely and just call next 2 lines with `.`.

**Review by sebastianst — APPROVED** [review]
🔥

File: ops/scripts/update-op-geth.py (hunk 1 above)
Comment (sebastianst) [reply ops/scripts/update-op-geth.py:13]: actually, we can just remove the whole `update-op-geth` script.

Comment (tynes) [reply ops/scripts/update-op-geth.py:13]: > actually, we can just remove the whole `update-op-geth` script.

want to follow up with this @sebastianst ?

**Review by tynes — APPROVED** [review]

**Review by hamdiallam — APPROVED** [review]

**Comment by mariuspod** [comment]
@mslipper
is there any reason this has been removed or has the indexer been migrated to a new repo instead ? It was pretty cool to have this for deposits/withdrawals and I'm wondering if there are any good alternatives now.

**Pull request merged**